cypress/e2e/modal-driver.ts

```diff
@@ -8,9 +8,9 @@ export default class ModalDriver {
     fillLayers: (opts: { type: string; layer?: string; id?: string }) => {
       // Having logic in test code is an anti pattern.
       // This should be splitted to multiple single responsibility functions
-      const type = opts.type;
+      const type: string = opts.type;
```

Comment:
I don't think this is needed in terms of typescript definitions (as opposed to two lines below where it is needed).